File: OPP/JS/driver/Driver.js
import Account from "../Account.js";

export default class Driver extends Account {
  constructor(name, document, email, password, carLicense) {
    super(name, document, email, password);
    this.carLicense = carLicense;
  }

  printDataDriver() {
    console.log(`Driver Name: ${this.name}`);
    console.log(`Driver ID: ${this.document}`);
    console.log(`Email: ${this.email}`);
    console.log(`Password: ${this.password}`);
    console.log(`Car license: ${this.carLicense}`);
  }
}

File: OOP-js-0/Java/Driver.java
class Driver extends Account {
  String carLicense;

  public Driver(String name, String document, String carLicense, String email, String password) {
    super(name, document, email, password);
    this.carLicense = carLicense;
  }

  void getDataDriver() {
    System.out.println("Driver Name: " + name);
    System.out.println("Driver ID: " + document);
    System.out.println("Car license: " + carLicense);
    System.out.println("Email: " + email);
    System.out.println("Password: " + password);
  }
}
